rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/OpenehrCompositionController.java

```diff
@@ -84,11 +83,13 @@
 
     @Autowired
     public OpenehrCompositionController(CompositionService compositionService) {
-
         this.compositionService = Objects.requireNonNull(compositionService);
     }
 
     @PostMapping(value = "/{ehr_id}/composition", consumes = {"application/xml", "application/json"})
+    // checkAbacPre /-Post attributes (type, auth object, subject, payload, content type)
+    @PreAuthorize("checkAbacPre(@openehrCompositionController.COMPOSITION, authentication, "
```

Comment:
This could be simplified based on the comment in`CustomMethodSecurityExpressionRoot`